libimage/manifest_list.go

```diff
@@ -210,8 +233,21 @@ func (i *Image) IsManifestList(ctx context.Context) (bool, error) {
 }
 
 // Inspect returns a dockerized version of the manifest list.
-func (m *ManifestList) Inspect() (*manifest.Schema2List, error) {
-	return m.list.Docker(), nil
+func (m *ManifestList) Inspect() (*InspectListInfo, error) {
```

Comment:
```suggestion
func (m *ManifestList) Inspect() (*ManifestListData, error) {
```

To be ~consistent with `ImageData` naming for inspecting an image.